Resources/Prototypes/_Nuclear14/Entities/Objects/Misc/junk.yml

```diff
@@ -216,6 +225,10 @@
     - JunkItem
     - Trash
     - CookPot
+  - type: WelderRefinable
+    refineResult:
+    - SheetSteel2
```

Comment:
There is no item called `SheetSteel2` All instances of material stacks default to 1 , 10 , 30 only

Reply:
Example of this would be `SheetSteel1` `SheetSteel10` `SheetSteel30`

Reply:
there's a way to let it drop two stacks of one steel each?